packages/wonder-blocks-dropdown/src/components/dropdown-core.js

```diff
@@ -332,30 +338,21 @@ class DropdownCore extends React.Component<Props, State> {
     // Figure out focus states for the dropdown after it has changed from open
     // to closed or vice versa
     initialFocusItem() {
-        const {
-            keyboard,
-            initialFocusedIndex,
-            open,
-            onSearchTextChanged,
-            searchText,
-        } = this.props;
+        const {open, onSearchTextChanged, searchText} = this.props;
 
         if (open) {
-            // Reset focused index
-            this.focusedIndex = initialFocusedIndex;
-            // We explicitly set focus to the first item only if we sense
-            // that the user opened the menu via the keyboard
-            if (keyboard) {
-                this.keyboardNavOn = true;
```

Comment:
There's other code in here that uses this value that would need to be addressed in this change as well once we have more clearly defined behavior